Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Mount /sys read-only in lxc-debian to prevent (one way of) escaping containers (Closes: #770901).

diff -Naurp lxc.orig/config/templates/debian.common.conf.in lxc/config/templates/debian.common.conf.in
--- lxc.orig/config/templates/debian.common.conf.in
+++ lxc/config/templates/debian.common.conf.in
@@ -3,7 +3,7 @@ lxc.pivotdir = lxc_putold
 
 # Default mount entries
 lxc.mount.entry = proc proc proc nodev,noexec,nosuid 0 0
-lxc.mount.entry = sysfs sys sysfs defaults 0 0
+lxc.mount.entry = sysfs sys sysfs ro 0 0
 
 # Default console settings
 lxc.tty = 4
